src/clr-angular/utils/drag-and-drop/droppable/droppable.ts

```diff
@@ -45,72 +45,39 @@ export class ClrDroppable<T> implements OnInit, OnDestroy {
     }
 
     private _group: string|string[];
-    private _dropTolerance: ClrDropTolerance = {top: 0, right: 0, bottom: 0, left: 0};
 
     @Input("clrGroup")
     set group(value: string|string[]) {
         this._group = value;
     }
 
+    private _dropTolerance: ClrDropTolerance = {top: 0, right: 0, bottom: 0, left: 0 };
+
+    private dropToleranceGenerator(top?: any, right?: any, bottom?: any, left?: any): ClrDropTolerance {
+        return {top: top, right: right || top, bottom: bottom || top, left: left || right || top };
```

Comment:
Oops, just realized this doesn't work. `"20 0"` to have only vertical tolerance will set everything to 20 because 0 is falsy.